searchobject/src/main/java/org/jalau/at18/searchobject/middleware/EmotionRecognitionMiddleware.java

```diff
@@ -97,18 +105,23 @@ else if (tokenCounter < 1) {
                 pw.flush(); //delete the content of the txt file.
                 pw.close();
                 fw.close();
-                throw new MiddlewareException("    Token has no more uses, please request another one");
+                throw new MiddlewareException(" Token has no more uses, please request another one");
             } else {
                 LOG.warning(" GENERATED TOKEN");
-                throw new MiddlewareException("    Generated token ");
+                throw new MiddlewareException(" Generated token ");
             }
         } catch (InstantiationError | MiddlewareException e) {
             LOG.warning(" ERROR LOADING THE MODEL " + e);
             e.printStackTrace();
-            PrintWriter out = response.getWriter();
             res.setStatus(400);
-            out.println ("    Status :    " + res.getStatus());
-            out.println (e.getMessage());
+            String x = Integer.toString( res.getStatus());
```

Comment:
Please rename this variable `x` to a name more descriptive

Reply:
variable name changed